src/NfcIpcSocket.cpp

```diff
@@ -90,34 +90,88 @@ int NfcIpcSocket::getListenSocket() {
   return nfcdConn;
 }
 
+int NfcIpcSocket::getConnectedSocket(const char* socketName)
+{
+  size_t len = strlen(socketName);
+  if (len > (SIZE_MAX - 2)) {
+    ALOGE("Socket address too long\n");
+    return -1;
+  }
+
+  size_t siz = len + 2;
+  if (siz > UNIX_PATH_MAX) {
+    ALOGE("Socket address too long\n");
+    return -1;
+  }
+
+  struct sockaddr_un addr;
+  addr.sun_family = AF_UNIX;
+  addr.sun_path[0] = '\0'; /* abstract socket namespace */
+  memcpy(addr.sun_path + 1, socketName, len + 1);
+  socklen_t addrLen = offsetof(struct sockaddr_un, sun_path) + siz;
+
+  int nfcdRw = socket(AF_UNIX, SOCK_STREAM, 0);
+  if (nfcdRw < 0) {
+    ALOGE("Could not create %s socket: %s\n", socketName, strerror(errno));
+    return -1;
+  }
+
+  int res = TEMP_FAILURE_RETRY(
+    connect(nfcdRw, reinterpret_cast<struct sockaddr*>(&addr), addrLen));
+  if (res < 0) {
+    ALOGE("Could not connect %s socket: %s\n", socketName, strerror(errno));
+    close(nfcdRw);
+    return -1;
+  }
+
+  return nfcdRw;
+}
+
 void NfcIpcSocket::setSocketListener(IpcSocketListener* listener) {
   mListener = listener;
 }
 
-void NfcIpcSocket::loop()
+void NfcIpcSocket::loop(const char* socketName)
```

Comment:
Recently i update a patch that function name will begin with upper case and all arguments will start with 'a', ex aSocketName to follow Mozilla C++ coding style.
Could you also help update the naming style, thanks